### ISSUE TITLE
I want to be able to complete setup and issue some commands without a currently running player

### ISSUE THREAD

**Issue by josephschmitt** [OverloadUT/alexa-plex#48]
Right now if you don't have Plex Home Theater open, the authentication and all commands fail. Obviously no play commands should work while HT is closed, but things like authenticating, asking what's on deck, and what's been recently added should not require an open player.

**Comment by OverloadUT** [comment]
Good catch - there's definitely some missing null checks throughout the code.
